### PR TITLE
Ticket/06_collapsibles-touch

### PR DESCRIPTION
**Pull request by marinarodriguez**
Modified collapsibles so they drop down when click directly on the panel and not only on the arrow.

Modified mainly on `collapsible.js`, where every listener and many constants have been changed to apply functions correctly: 
-Found 3 constants (couldn't find them in other JavaScript files) that could to be removed.
-Also the 3 constants for the arrows at the start of the file could be removed too.

Added a class in every `panel-dropdown` in `_share.html`, `_fillApp.html` and `_design.html` to apply listeners on them. The last one has been indented just to see hierarchy clearly.

Please check that everything works fine!

### CODE REVIEW

File: _src/assets/js/collapsible.js [hunk 1]
```diff
@@ -9,38 +9,42 @@ const arrowElDesign = document.getElementById("dropdown__down-arrow--design");
...
+// TO BE REMOVED -- Apparently these 3 could be removed. Need approval to do so.
+const panelDesignEl = document.querySelector(".panel__design-radios");
 const panelFillEl = document.querySelector(".panel-fill__form");
 const panelShareEl = document.querySelector(".align_share");
 
+// Consts for listeners
+const designDropdownEl = document.querySelector(".panel__dropdown--design");
+const fillDropdownEl = document.querySelector(".panel__dropdown--fill");
+const shareDropdownEl = document.querySelector(".panel__dropdown--share");
+
```
Comment (irene-gomez) [inline _src/assets/js/collapsible.js]: I think `panelDesignEl`, `panelFillEl` and `panelShareEl` could be made with a `querySelectorAll('panel__dropdown')` and go through them with a `for... of`. And this way we save ourselves from creating more constants.

Comment (marinarodriguez) [reply _src/assets/js/collapsible.js]: On it!

Comment (marinarodriguez) [reply _src/assets/js/collapsible.js]: Should be fixed!

Comment (irene-gomez) [reply _src/assets/js/collapsible.js]: Perfect. You can merge

**Pull request merged**